roofit/roostats/inc/RooStats/ToyMCSampler.h

```diff
@@ -249,7 +249,7 @@ class ToyMCSampler: public TestStatSampler {
 
       // densities, snapshots, and test statistics to reweight to
       RooAbsPdf *fPdf; // model (can be alt or null)
-      const RooArgSet* fParametersForTestStat;
+      std::unique_ptr<const RooArgSet> fParametersForTestStat;
```

Comment:
Does ROOT know automatically how to do the schema evolution from raw pointer to `std::unique_ptr`?

Reply:
Yes :).  You might still need to update the ClassDef  version number if any.

Reply:
Thanks, good to know!

Reply:
I was hoping that Philippe would say that. :-)